classification/src/main/java/dev/ikm/komet/reasoner/ReasonerResultsNode.java

```diff
@@ -76,47 +81,61 @@ public ReasonerResultsNode(ViewProperties viewProperties, KometPreferences nodeP
 					activityStreamKeyProperty, optionForActivityStreamKeyProperty, centerBox);
 			this.contentPane.setTop(topPanelParts.topPanel());
 			Platform.runLater(() -> {
-				ArrayList<MenuItem> collectionMenuItems = new ArrayList<>();
-				collectionMenuItems.add(new SeparatorMenuItem());
-
+				ArrayList<MenuItem> menuItems = new ArrayList<>();
+				ArrayList<CheckMenuItem> reasonerServiceMenuItems = new ArrayList<>();
+				menuItems.add(new SeparatorMenuItem());
 				List<ReasonerService> rss = PluggableService.load(ReasonerService.class).stream().map(Provider::get)
 						.sorted(Comparator.comparing(ReasonerService::getName)).toList();
 				for (ReasonerService rs : rss) {
 					LOG.info("Reasoner service add: " + rs);
-					MenuItem item = new MenuItem("Use " + rs.getName());
-					item.setOnAction(x -> {
+					CheckMenuItem item = new CheckMenuItem("Use " + rs.getName());
+					item.setOnAction(ae -> {
 						this.reasonerService = rs;
+						reasonerServiceMenuItems.forEach(xi -> xi.setSelected(false));
+						item.setSelected(true);
 						LOG.info("Reasoner service selected: " + rs.getName());
 					});
-					collectionMenuItems.add(item);
-					if (this.reasonerService == null)
+					menuItems.add(item);
+					reasonerServiceMenuItems.add(item);
+					if (this.reasonerService == null) {
 						this.reasonerService = rs;
+						item.setSelected(true);
+					} else if (rs.getName().equals("ElkSnomedReasonerService")) {
```

Comment:
Can we make literal Strings as constants? magic values: "ElkSnomedReasonerService"?